src/firmware/app/primitives/move_helper.c

```diff
@@ -199,7 +197,7 @@ void app_move_helper_tick(void* void_state_ptr, FirmwareWorld_t* world)
 
     // plan major axis movement
     float max_major_a     = 3.5;
-    float max_major_v     = state->slow ? 1.25 : 3.0;
+    float max_major_v     = 3.0;
```

Comment:
What does `max_major_v` or `max_maxjor_a` stand for?

Reply:
max velocity in the major axis direction